notebooks/estimate_resource_usage.ipynb

```diff
@@ -0,0 +1,591 @@
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "## Load Inputs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import cromwell_tools as cwt\n",
+    "from google.cloud import storage\n",
+    "import glob\n",
+    "from collections import defaultdict\n",
+    "from scsequtil.plot import grid\n",
+    "from operator import itemgetter\n",
+    "from sklearn.linear_model import LinearRegression\n",
+    "import numpy as np\n",
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "google_project = 'insert_your_project'\n",
+    "cromwell_url = 'your_cromwell_url'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "## Configure Cloud Entities\n",
+    "\n",
+    "Configure a connection to a locally initiated version of cromwell (version >= 29) and the google project. You will likely have to authenticate independently to enable access to your google storage. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "local_config = {'cromwell_url': cromwell_url}\n",
+    "cromwell = cwt.Cromwell(**local_config)\n",
+    "cromwell.server_is_running()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "client = storage.Client(project=google_project)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "## Set up testing workflows\n",
+    "\n",
+    "in the `src/cromwell_tools/test/data/` folder are four WDL files pointing to google storage locations that each contain a different number of fastq reads: 10,000, 100,000, 1,000,000, and 10,000,000. We will use a monitoring script to run each of these workflows, and then calculate the resource usage (memory and disk) of each file throughout each of the run tasks"
```

Comment:
Where can I find the `src/cromwell_tools/test/data/` folder?

Reply:
That's unfortunately. I named the python package I made to run and monitor cromwell jobs `cromwell_tools`. 
`https://github.com/ambrosejcarr/cromwell_tools`

... this package is also called cromwell-tools. 

I'm happy to change the name of my repo.

Reply:
Ok! Yeah, the name makes it seem like it's your fork of this repository. Can you clarify how it is different?

Reply:
Also is the a particular reason why the example data is in a separate repo?

Reply:
I coded up the repo in question to launch, track, and debug cromwell runs, because we currently lack a GUI/API to do it, and it was annoying to have to go back and forth between swagger, browser, github, and google storage. [This notebook](https://github.com/ambrosejcarr/cromwell_tools/blob/master/src/examples/basic_functionality.ipynb) shows most of the functionality. 

It will likely be superseded by the job manager, when that's up and running. I can rename it cromwell-manager, for now. 

Regarding the data separation, I initially had the notebook in that repo. I will add a `data/` directory to this repo and put any dependencies there.

Reply:
Sounds good! Thanks for the clarification!